composer.json

```diff
@@ -23,43 +23,42 @@
         }
     ],
     "require": {
-        "php": "^7.2 || ^8.0",
+        "php": ">=7.3 || ^8.0",
         "ext-curl": "*",
         "ext-json": "*",
         "ext-simplexml": "*",
-        "symfony/process": "~3.4 || ~4.0 || ~5.0",
+        "symfony/process": "~3.4 || ~4.0 || ~5.0 || ~6.0",
```

Comment:
Should also update the minimum supported versions of these libraries based on our new minimum PHP requirement